turbo.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://turborepo.com/schema.json",
+  "tasks": {
+    "build": {
+      "dependsOn": ["^build"],
+      "cache": false
+    },
+    "@ni-private/blazor-workspace#build": {
+      "dependsOn": ["^build", "^build-bundle"]
+    },
+    "build-bundle": {
+      "dependsOn": ["build"],
+      "cache": false
+    },
+    "test": {
+      "cache": false
+    },
+    "lint": {
+      "cache": false
+    },
```

Comment:
The test and lint commands implicitly depend on the build command of the same package, correct? What enforces that? Based on [Depending on tasks in the same package](https://turborepo.com/docs/crafting-your-repository/configuring-tasks#depending-on-tasks-in-the-same-package) I was expecting to see `"dependsOn": ["build"]` on these.